test-app/tests/unit/sinon-sandbox-test.ts

```diff
@@ -1,9 +1,6 @@
 import { module, test } from 'qunit';
 import sinon from 'sinon';
-import {
```

Comment:
looks safe to me as those are not public API?

Reply:
@SergeAstapov from what I can see they are used as test helpers for the test app, so probably is fine?